spinnaker-jenkins-to-vmss/scripts/set_azure_spinnaker.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Get azure data 
+# clientId : -c 
+# AppKey: -a
+# Default values
+JENKINS_USERNAME=""
+JENKINS_PASSWORD=""
+
+while getopts ":t:s:p:c:h:r:l:k:u:q:f:a:i:" opt; do
+  case $opt in
+    t) TENANTID="$OPTARG"
+    ;;
+    p) PASSWORD="$OPTARG"
+    ;;
+    c) CLIENTID="$OPTARG"
+    ;;
+    s) SUBSCRIPTIONID="$OPTARG"
+    ;;
+    h) PACKERSTORAGEACCOUNT="$OPTARG"
+    ;;
+    r) RESOURCEGROUP="$OPTARG"
+    ;;
+    l) RESOURCEGROUPLOCATION="$OPTARG"
+    ;;
+    k) KEYVAULT="$OPTARG"
+    ;;
+    i) JENKINS_FQDN="$OPTARG"
+    ;;
+    u) JENKINS_USERNAME="$OPTARG"
+    ;;
+    q) JENKINS_PASSWORD="$OPTARG"
+    ;;
+    f) FRONT50_STORAGE="$OPTARG"
+    ;;
+    a) FRONT50_KEY="$OPTARG"
+    ;;
+  esac
+done
+
+WORKDIR=$(pwd)
+# Usually the workdir is /var/lib/waagent/custom-script/download/0
+JENKINS_URL='http:\/\/'$JENKINS_FQDN
+DEBIAN_REPO='http:\/\/ppa.launchpad.net\/openjdk-r\/ppa\/ubuntu trusty main;'$JENKINS_URL
+DEBUG_FILE=$WORKDIR"/debugfile"
```

Comment:
We used this file during the development mode, it is not needed for the publication